DBlock Game/DBlock Game/Program.cs

```diff
@@ -1347,6 +1355,133 @@ public static void Fight()
             Menu();
         }
 
+        public static void Death()
+        {
+            Random rand = new Random();
+            string player;
+            string comp;
+            int pscore = 0;
+            int cscore = 0;
+
+            // the paper,scissors,rock game
+
+            for (int i = 0; i < 4; i++)
+            {
+                player = "";
+                comp = "";
+
+                // while loop that check the inputs the values are clear
+
+                while (player != "PAPER" && player != "SCISSORS" && player != "ROCK")
+                {
+                    TextBox("you have to play paper,scisors,rock to win its a best out of 3.", Player, false);
+                    TextBox("Press 1 for paper, Press 2 for scissors, Press 3 for rock.", Player,true);
+
+                    string temp = Console.ReadLine();
+                    switch (temp)
+                    {
+                        case "1":
+                            player = "PAPER";
+                            break;
+                        case "2":
+                            player = "SCISSORS";
+                            break;
+                        case "3":
+                            player = "ROCK";
+                            break;
+                    }
+
+                }
+
+                // the computers choice
+                int num2 = rand.Next(1, 4);
```

Comment:
You could have a constant in for the random.next if u wanted, and it could have been rand.next(3); instead of (1,4)